### ISSUE TITLE
py3 support

### ISSUE THREAD

**Issue by sagdusmir** [ndparker/tdi#4]
Support for a more recent version of python would be appreciated. I just came across the PyConDE session and found py3 being mentioned there....  *sigh*

**Comment by ndparker** [comment]
Yes... started some time ago here: https://github.com/ndparker/tdi3

Admittedly it's not very far yet, but it's a start :-)
